I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/gameintro.rpy", line 10, in script
    show abbe at abbe_spot
  File "game/gameintro.rpy", line 10, in <module>
    show abbe at abbe_spot
NameError: name 'abbe_spot' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/gameintro.rpy", line 10, in script
    show abbe at abbe_spot
  File "C:\Users\Owner\Desktop\renpy-8.1.3-sdk\renpy\ast.py", line 1428, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\Owner\Desktop\renpy-8.1.3-sdk\renpy\ast.py", line 1387, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\Owner\Desktop\renpy-8.1.3-sdk\renpy\ast.py", line 1387, in <listcomp>
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\Owner\Desktop\renpy-8.1.3-sdk\renpy\python.py", line 1153, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Owner\Desktop\renpy-8.1.3-sdk\renpy\python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/gameintro.rpy", line 10, in <module>
    show abbe at abbe_spot
NameError: name 'abbe_spot' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
The House On Limbo Lane 1.0
Sun Oct  1 23:34:52 2023
